### ISSUE TITLE
Auth call fails with 404

### ISSUE THREAD

**Issue by renkoh** [CharlesGillanders/alphaess#17]
Calls to the `/Account/Login` endpoint fail with response:

```
{'code': 6026, 'msg': '404 NOT_FOUND'}
```

it seems that the API may have changed completeley

**Comment by CharlesGillanders** [comment]
> Calls to the `/Account/Login` endpoint fail with response:
> 
> 
> 
> ```
> 
> {'code': 6026, 'msg': '404 NOT_FOUND'}
> 
> ```
> 
> 
> 
> it seems that the API may have changed completeley
> 
> 

Yes AlphaEss have asked everyone to switch to their new OpenAPI - I have done so for my Home Assistant integration using a new python library which is also available on GitHub and which is intended to replace this one.

[https://github.com/CharlesGillanders/alphaess-openAPI](https://github.com/CharlesGillanders/alphaess-openAPI)
